packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -123,6 +135,27 @@ function getPresetsClasses( blockSelector, blockPresets = {} ) {
 	);
 }
 
+const PresetSvgFilter = withFilters( 'editor.PresetSvgFilter' )( () => null );
```

Comment:
What's the value of creating a specific filter for rendering duotone in the site editor? This is the reasons I see:

- We plan that third-parties will can hook into it. I don't understand how/why this would be the case?
- It's a way to share behaviour between the block-editor and the edit-site packages. If this is the primary reason, can't we just export the function we need (`DuotoneFilter`) from the block-editor package and import it here, as we do with the rest of the hook utilities?

Reply:
> We plan that third-parties will can hook into it.

This is the long-term vision of it—plugin authors being able to create their own complex filters that don't make sense for core. But maybe I'm getting ahead of myself and should wait until the other pieces of that are more complete. 😅

Reply:
Ah, I see. Yeah, I think we can go with the import solution and leave the filter out for when we cross that bridge.